Name_Generator_lib/NameGen_Main.cpp

```diff
@@ -1,9 +1,18 @@
 #define _CRT_SECURE_NO_WARNINGS
 
+#ifdef __APPLE__
+
+#include <stdlib.h>
+#include <malloc/malloc.h>
+
+#elif __linux
```

Comment:
This looks like it will break Windows build. Using `#else` would be just enough.

Reply:
Done